src/openrct2-ui/windows/Map.cpp

```diff
@@ -162,7 +163,8 @@ class MapWindow final : public Window
         CentreMapOnViewPoint();
         FootpathSelectDefault();
 
-        _mapWidthAndHeightLinked = gMapSize.x == gMapSize.y;
+        auto& gameState = OpenRCT2::GetGameState();
```

Comment:
Should I change to use a namespace as OpenRCT2 is referenced a lot in this file?

Reply:
Yeah give it a try, there might be a problem but when it doesn't cause any compiler errors then its actually good as we had planned to move everything into the OpenRCT2 namespace eventually.